hudi-client/hudi-spark-client/src/main/java/org/apache/hudi/table/action/bootstrap/SparkBootstrapCommitActionExecutor.java

```diff
@@ -320,18 +314,8 @@ private Map<BootstrapMode, List<Pair<String, List<HoodieFileStatus>>>> listAndPr
     BootstrapModeSelector selector =
         (BootstrapModeSelector) ReflectionUtils.loadClass(config.getBootstrapModeSelectorClass(), config);
 
-    Map<BootstrapMode, List<String>> result = new HashMap<>();
-    // for FULL_RECORD mode, original record along with metadata fields are needed
-    if (FULL_RECORD.equals(config.getBootstrapModeForRegexMatch())) {
```

Comment:
@codope Here we don't need to do any extra conversion, because FULL_RECORD is configured when using `BootstrapRegexModeSelector`